.github/workflows/documentation-creation.yml

```diff
@@ -0,0 +1,50 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Doxygen Action
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ main ]
```

Comment:
Here to change to "master", as my temporary created repo has used the new "main" as default.